tests/flow/test_path.py

```diff
@@ -0,0 +1,142 @@
+import os
+import sys
+from redisgraph import Graph, Node, Edge, Path
+from demo import QueryInfo
```

Comment:
Delete line, this import is performed below (after appending to path).

Reply:
done